bindings/python/iota_sdk/types/unlock.py

```diff
@@ -88,6 +89,11 @@ class NftUnlock:
     type: int = field(default_factory=lambda: int(UnlockType.Nft), init=False)
 
 
+# pylint: disable=missing-function-docstring,unused-argument
+def deserialize_unlocks(dicts: List[Dict[str, Any]]) -> List[Unlock]:
```

Comment:
Why do we need this extra?

Reply:
because of the real impl needing the MultiUnlock already, but that MultiUnlock also needed deserialize_unlocks and so on. So this way the fn is defined and impl later. Or do you know a better way?

Reply:
One could maybe move it in another file, but probably not really better
Let's just add a comment why this is done here